### PR TITLE
Fix unused param warning in NetworkStorageManager

### PR DESCRIPTION
**Pull request by ykimot**
#### 1a1b5cfdbcfae48185468d9a751ea0ad377bfe44
<pre>
Fix unused param warning in NetworkStorageManager
<a href="https://bugs.webkit.org/show_bug.cgi?id=255436">https://bugs.webkit.org/show_bug.cgi?id=255436</a>

Reviewed by Chris Dumez.

If RELEASE_LOG() is desiabled, `deletedOriginCount` is not used in
NetworkStorageManager::performEviction() because &apos;deletedOriginCount`
is only used for a debug print purpose. Though `deletedOriginCount` is
incremented, compilers&apos; optimization omits the variable.

* Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp: Add UNUSED_PARAM for `deletedOriginCount` next to the RELEASE_LOG.
(WebKit::NetworkStorageManager::performEviction):

Canonical link: <a href="https://commits.webkit.org/262946@main">https://commits.webkit.org/262946@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/2253e44e831c7b0525aa1e719bbf097522f275e5

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/3143 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/3203 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/3313 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/4555 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/3501 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/3264 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/3230 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/2733 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/3174 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/3491 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/2824 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/4361 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/989 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/2799 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/2637 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/2766 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/2850 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/4103 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/3218 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/2567 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/2803 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/2801 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/759 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/2795 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/3062 "Built successfully") | | |

### CODE REVIEW

**Review by cdumez — APPROVED** [review]

File: Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp [hunk 1]
```diff
@@ -1798,4 +1799,3 @@ bool NetworkStorageManager::shouldManageServiceWorkerRegistrationsByOrigin()
 #endif // ENABLE(SERVICE_WORKER)
 
 } // namespace WebKit
-
```
Comment (rkirsling) [inline Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp:1801]: Please drop this line from the patch.

**Review by rkirsling — APPROVED** [review]
r=me

File: Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp (hunk 1 above)
Comment (cdumez) [inline Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp:1801]: It's an extra line. I think it's OK to remove it in this patch this time, though we usually try to avoid unrelated spacing changes in patches.

Comment (rkirsling) [reply Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp:1801]: Yeah, sorry about that. I deleted my comment once I realized that there was in fact an extra line at the end.

Comment (ykimot) [reply Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp:1801]: > though we usually try to avoid unrelated spacing changes in patches.

I'll keep in mind.
